### DISCUSSION TITLE
ModuleNotFoundError: No module named 'torchvision.models.efficientnet'

### DISCUSSION THREAD

**Discussion by LiJiaming5558** [ultralytics/yolov5#10062] [Q&A]
ModuleNotFoundError: No module named 'torchvision.models.efficientnet'
classify/train.py

Traceback (most recent call last):
  File "D:/lijaiming/ai3/yolov5-master/classify/train.py", line 358, in <module>
    main(opt)
  File "D:/lijaiming/ai3/yolov5-master/classify/train.py", line 344, in main
    train(opt, device)
  File "D:/lijaiming/ai3/yolov5-master/classify/train.py", line 129, in train
    model = attempt_load(opt.model, device='cpu', fuse=False)
  File "D:\lijaiming\ai3\yolov5-master\models\experimental.py", line 79, in attempt_load
    ckpt = torch.load(attempt_download(w), map_location='cpu')  # load
  File "D:\ProgramData\Miniconda3\lib\site-packages\torch\serialization.py", line 594, in load
    return _load(opened_zipfile, map_location, pickle_module, **pickle_load_args)
  File "D:\ProgramData\Miniconda3\lib\site-packages\torch\serialization.py", line 853, in _load
    result = unpickler.load()
ModuleNotFoundError: No module named 'torchvision.models.efficientnet'

**Comment by glenn-jocher** [comment]
👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃
